### PR TITLE
Update iOS SDK v2.1.10 release notes

### PR DESCRIPTION
**Pull request by vonageCI**
iOS SDK v2.1.10 notes added to `release-notes.md`.

### CODE REVIEW

**Comment by horothesun** [comment]
⚠️ Please, merge after https://github.com/Nexmo/nexmo-developer/pull/2672 ⚠️

File: _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ navigation_weight: 0
 
 # Release Notes
 
+## 2.1.10 - 2020-04-16
```
Comment (tbedford) [inline _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md:9]: New line after heading.

File: _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ navigation_weight: 0
...
+### Internal
```
Comment (tbedford) [inline _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md:10]: New line after heading.

File: _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md (hunk 1 above)
Comment (horothesun) [reply _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md:9]:
```suggestion
## 2.1.10 - 2020-04-16

```

File: _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md (hunk 2 above)
Comment (horothesun) [reply _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md:10]:
```suggestion
### Internal

```

**Comment by horothesun** [comment]
Sorry for the newlines, I assume there was a copy-paste problem from a code block section in a Slack message: it looks like the newlines get removed.

**Pull request merged**